model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -1351,7 +1372,7 @@ async def execute(
         )
 
         if len(model_endpoints) == 0:
-            raise ObjectNotFoundException(f"Model endpoint {model_endpoint_name} not found.")
+            raise ObjectNotFoundException
```

Comment:
Was this change intentional?

Reply:
Oops thanks for the catch, not sure how that got deleted, will fix